<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankingFrame</class>
 <widget class="QFrame" name="BankingFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1270</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #212529;
border: 0px;
color: #aaaaaa;


font-size: 15px;
font-family: Lato;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; 
font-size: 22px;</string>
       </property>
       <property name="text">
        <string>BANKING</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_withdrawButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#m_withdrawButton {color: orange; background-color: #212529; border: 1px solid orange; font-size: 16px;}

QPushButton#m_withdrawButton:hover {color: orange; background-color: #212529; border: 2px solid orange; font-size: 16px;}</string>
       </property>
       <property name="text">
        <string>WITHDRAW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_backButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#m_backButton {color: #ccc; border: 1px solid #343a40; font-size: 16px; background-color: #212529;}

QPushButton#m_backButton:hover {color: #ddd; border: 2px solid #444; font-size: 16px; background-color: #212529;}</string>
       </property>
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="m_unlockedInvestmentLabel">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; font-size: 14px; background-color: #333;</string>
       </property>
       <property name="text">
        <string notr="true">UNLOCKED INVESTMENTS 0.00 CCX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_unlockedDepositLabel">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; font-size: 14px; background-color: #333;</string>
       </property>
       <property name="text">
        <string notr="true">UNLOCKED DEPOSITS 0.00 CCX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="m_depositView">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
background-color: #282d31; 
   color: #ddd;
	font-family: Lato;
    border: 0px solid white;
font-size: 14px;
border-bottom: 1px solid orange;
   height: 30px;

}

QTreeView::item {
    color: #ddd;
    height: 35px;
}

QTreeView::branch:selected {
  background-color: orange;
  color: black;
}

QWidget:item:selected {
  font-family: Lato;
  background-color: #282d31;
  color: #ddd;
}

margin-top: 20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="indentation">
      <number>5</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_withdrawButton</sender>
   <signal>clicked()</signal>
   <receiver>BankingFrame</receiver>
   <slot>withdrawClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_depositView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>BankingFrame</receiver>
   <slot>showDepositDetails(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backButton</sender>
   <signal>clicked()</signal>
   <receiver>BankingFrame</receiver>
   <slot>backClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1195</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>depositClicked()</slot>
  <slot>investmentClicked()</slot>
  <slot>withdrawClicked()</slot>
  <slot>showDepositDetails(QModelIndex)</slot>
  <slot>timeChanged(int)</slot>
  <slot>timeChanged2(int)</slot>
  <slot>investmentsClicked()</slot>
  <slot>allButtonClicked()</slot>
  <slot>depositsClicked()</slot>
  <slot>depositParamsChanged()</slot>
  <slot>investmentParamsChanged()</slot>
  <slot>backClicked()</slot>
 </slots>
</ui>
